cost/superseded_instance/superseded_instance.pt

```diff
@@ -396,10 +396,16 @@ end
 # Escalations
 ###############################################################################
 escalation "report_instances" do
+  automatic true
+  label "Send Mail"
+  description "Sends incidents email"
   email $param_email
 end
 
 resolution "report_and_resize_instances_resolution" do
+  automatic false
+  label "Resize Instances"
```

Comment:
The resolution doesn't need to be changes.  your update probably doesn't have an effect.  Also, the label really doesn't match what the resolution does.

Reply:
reverted resolution to the original state.